src/System.Management.Automation/engine/parser/TypeResolver.cs

```diff
@@ -756,6 +756,7 @@ internal static class CoreTypes
                     { typeof(PSListModifier),                              new[] { "pslistmodifier" } },
                     { typeof(PSObject),                                    new[] { "psobject", "pscustomobject" } },
                     { typeof(PSPrimitiveDictionary),                       new[] { "psprimitivedictionary" } },
+                    { typeof(PSPropertyExpression),                        new[] { "pspropertyexpression"} },
```

Comment:
@PaulHigin I would like you to take another look at this change. `PSPropertyExpression` is added to `CoreTypes`, and that means users can call its methods in `ConstrainedLanguage` mode. Do you have any concerns about it?

Reply:
Thanks, I didn't see that.  If this type is going to be white listed then it needs to go through a separate/full security review.  I see that some internal methods are now public, and these along with constructors need to be reviewed for vulnerabilities.  Please set up a meeting for this.

Reply:
No - it's only intended to be a type accelerator. I added it in the wrong place. I'll move it to the `TypeAccelerators()` method where it should be.

Reply:
Yeah, I agree it should be moved :)